sample/development/components/MainScene.brs

```diff
@@ -57,45 +66,71 @@ sub init()
   '     "key2": "value2"
   '   }
   ' })
+end sub
 
+sub _sendEventWithCallback()
+  '----------------------------------------
+  ' Send an Experience Event with callback
+  '----------------------------------------
+
+  m.adobeEdgeSdk.sendEvent({
+    "eventType": "commerce.orderPlaced",
+    "commerce": {
+      "key3": "value3"
+    }
+  }, sub(context, result)
+    print "callback result: "
+    print result
+    print context
+    jsonObj = ParseJson(result.message)
+    message = ""
+    for each item in jsonObj.handle
+      if item.type = "locationHint:result" then
+        for each data in item.payload
+          if data.scope = "EdgeNetwork" then
+            message = "locationHint:EdgeNetwork: " + data.hint
+          end if
+        end for
+      end if
+    end for
+
+    ' show result in dialog
+    context.Warning.visible = "true"
+
+    context.Warning.message = message
+  end sub, m)
 end sub
 
-sub onButtonSelected()
-  'SendEvent button pressed
-  if m.ButtonGroup.buttonSelected = 0
-    '----------------------------------------
-    ' Send an Experience Event with callback
-    '----------------------------------------
+sub _testShutdownAPI()
+  if m.adobeEdgeSdk = invalid
```

Comment:
should we break this down to

setupSDK()
configure()
clearConfiguration()
SendEvent()
SendEventWithCallback()
shutDown()

this way we can clear the config and keep calling sendevent whcih will queue the events and call shutdown to test the scenario.

What do you think?

Reply:
I intend to keep the valid configuration and let the SDK task node (thread) keep sending edge requests. Then we can verify if the shutdown operation can deactivate the task node and drop requests correctly.